### PR TITLE
[Performance] Make performance tests consistent with multi-cluster

### LINKED ISSUE #6119: Enhance Meshery performance to account for multi-cluster deployments
#### Current Behavior

Currently Meshery performance tests require you to have a kubernetes cluster running. The behaviour of the tests is also undefined in case the user has multiple clusters running

#### Desired Behavior

If the user has no K8s clusters running, skip retrieving cluster info in tests. But still run the tests
If the user has multiple clusters running, retrieve enviroment information about all the clusters.

#### Mockups

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by gr455**
**Description**
This PR is an effort to make performance tests consistent with multi-cluster.

This PR fixes #6119

TODO:
- [x] Allow running tests without requiring k8s
- [x] Allow selecting contexts while running perf tests 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Review by leecalcote — DISMISSED** [review]
LGTM.

@gyohuangxin @hershd23 you might want to verify this change.

**Comment by gr455** [comment]
@leecalcote Ready to merge?

**Comment by leecalcote** [comment]
@gr455, yes. Although, we haven't heard back from @gyohuangxin or @hershd23, which is what I was waiting for...

File: server/handlers/load_test_handler.go [hunk 1]
```diff
@@ -364,22 +364,40 @@ func (h *Handler) executeLoadTest(ctx context.Context, req *http.Request, profil
...
 
 	mk8sContexts, ok := req.Context().Value(models.KubeClustersKey).([]models.K8sContext)
-	if !ok || len(mk8sContexts) == 0 {
-		h.log.Error(ErrInvalidK8SConfig)
-		return
+	if !ok {
+		h.log.Warn(ErrInvalidK8SConfig)
 	}
 	var wg sync.WaitGroup
+	h.log.Info(mk8sContexts)
```
Comment (hershd23) [inline server/handlers/load_test_handler.go:371]: Does this log any secrets by any chance? Ignore if it does not

**Review by hershd23 — COMMENTED** [review]
One comment rest LGTM

File: server/handlers/load_test_handler.go (hunk 1 above)
Comment (gr455) [reply server/handlers/load_test_handler.go:371]: It will. Thanks for pointing that out, we don't need that log anyways.

**Review by leecalcote — APPROVED** [review]

File: server/handlers/load_test_handler.go (hunk 1 above)
Comment (leecalcote) [reply server/handlers/load_test_handler.go]: Thanks, @hershd23 👏

**Pull request merged**